package.json

```diff
@@ -52,6 +52,7 @@
     "eslint": "^6.8.0",
     "jest": "^25.1.0",
     "metro-react-native-babel-preset": "^0.58.0",
+    "prettier": "1.17.0",
```

Comment:
Is this and the package-json intentional in this PR?  If not, let's remove these files.

Reply:
This was intentional - realized I should've included it in the primary Prettier PR, since different versions of Prettier have slightly different rules. I can make it a separate PR if you need